html/guess.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+
+    </head>
+    <body>
+        <p>Enter a Number</p>
+        <input id="num" type="number">
+        <button onclick="check()" >enter</button>
+       <p id="demo"></p>
+        <script>
+            function check()
+        {
+        var x = document.getElementById("num").value;
+            var y= Math.floor(Math.random()*100)+1;

```

Comment:
y will always change. move it out side the function. otherwise it will be very difficult to guess the number.